### DISCUSSION TITLE
Plusieurs Comptes Utilisateurs & Synapps

### DISCUSSION THREAD

**Discussion by FlorentAdell** [witsa/synapps#575] [Q&A] [answered]
Bonjour,

J'ai un petit soucis, j'ai plusieurs Synapps dans un Redy et plusieurs utilisateurs.
Dans l'idéal, j'aimerais que certains utilisateurs aient accès à une synapps  Synapps 1 et d'autres la deuxième Synapps

Exemple : Utilisateurs 1,2,3 = Synapps 1 mais pas la 2 
Exemple : Utilisateurs 4,5,6 = Synapps 2 mais pas la 1

![image](https://user-images.githubusercontent.com/46195964/226545217-683b6cdd-2e05-4b2b-8330-f582d0a7c764.png)

**Comment by WITNicolas** [comment, marked as answer]
Salut,
Au niveau de l'UTILISATEUR dans les préférences, URL par défaut, tu peux mettre le chemin de la SYNAPP.
Du style : /easy/synapps/maSynapp1

(maSynapp 1 étant le label de ta Synapp)
